tests/virtualization/universal/save_and_restore.pm

```diff
@@ -35,24 +37,27 @@ sub run_test {
         }
     }
 
+    # Starting all guests to create a test file in them, the file must not exist after restoring.
```

Comment:
For future test cases, we might want to create a separate subroutine within this test case to de-clutter `run_test` a bit. That could incorporate lines 40-51.